srml/council/src/motions.rs

```diff
@@ -30,7 +30,7 @@ pub type ProposalIndex = u32;
 
```

Comment:
suggest declare something like the following:
```
pub type CondonersCount = u32;
pub type TotalCount = u32;
```

Reply:
They're actually indistinguishable; both are just a count of members. Introducing that type is not an unreasonable change though.